Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -102,9 +102,30 @@ Ref<HTMLImageElement> HTMLImageElement::create(const QualifiedName& tagName, Doc
 HTMLImageElement::~HTMLImageElement()
 {
     document().removeDynamicMediaQueryDependentImage(*this);
+    setForm(nullptr);
+}
 
+HTMLFormElement* HTMLImageElement::form() const
+{
+    return m_form.get();
+}
```

Comment:
Put this as an inline in the header?

Reply:
Sure, why not.

Reply:
Looks like we can't do that without including HTMLFormElement.h in HTMLImageElement.h so perhaps not.

Reply:
Oh, right, because it’s a `WeakPtr`. For `RefPtr` it would not require that.